homeassistant/components/media_player/kodi.py

```diff
@@ -301,4 +301,4 @@ def media_seek(self, position):
 
     def play_media(self, media_type, media_id, **kwargs):
         """Send the play_media command to the media player."""
-        self._server.Player.Open({media_type: media_id}, {})
+        self._server.Player.Open({"item": {"file": str(media_id)}})
```

Comment:
Actually the file abstraction layer "resolves" the media type, should it be an url (http://myfavoritewebcast.xyz, plugin://plugin.video.youtube/?action=play_video&videoid=123abc) or any reachable path

Please, have a look at [http://kodi.wiki/view/JSON-RPC_API/#Player.Open](http://kodi.wiki/view/JSON-RPC_API/#Player.Open) for clear and comprehensible English specifications.

I'd love to elaborate a more flexible solution collaboratively.